micro-ui/web/micro-ui-internals/packages/modules/core/MIGRATION.md

```
@@ -0,0 +1,47 @@
+        "errorMessage": "CORE_COMMON_MOBILE_ERROR"
+      },
+      "attributes": {
+        "prefix": "+91"
+      }
+    }
+    // ... other field validations
+  ]
+}
+```
+
+### Affected Modules & Files
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)